docs/operations/mediated_devices_configuration.md

```diff
@@ -0,0 +1,121 @@
+# Automatic configuration of mediated devices / vGPUs
+
+KubeVirt aims to facilitate the configuration of mediated devices on large clusters.
+The new `mediatedDevicesConfiguration` API in KubeVirt CR allows adimnistrators to
+create or remove mediated devices in a declative way, by providing a list of the desired mediated devices types  that they expect to be configured in the cluster.
+
+It would also be possible to provide a more specific configuration that targets one or a group of nodes directly.
```

Comment:
I think we also explicitly mention the following to provide more clarity,
1)  `nodeMediatedDeviceTypes` needs to be used along with `mediatedDevicesTypes`, to override the global config brought in by, `mediatedDevicesTypes`.

2) Usage of only `nodeMediatedDeviceTypes` is not supported.